readthedocs/organizations/views/private.py

```diff
@@ -97,15 +99,25 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class AddOrganizationOwner(PrivateViewMixin, OrganizationOwnerView, CreateView):
-    template_name = 'organizations/admin/owners_edit.html'
-    success_message = _('Owner added')
+class AddOrganizationOwner(PrivateViewMixin, OrganizationOwnerView, FormView):
+    template_name = "organizations/admin/owners_edit.html"
+    success_message = _("Invitation sent")
+
+    def form_valid(self, form):
+        form.save()
+        return super().form_valid(form)
```

Comment:
This called my attention. Why are we saving the form _before_ validating it?

Reply:
`form_valid` is called only if the form is valid.

Reply:
Doesn't the `super().form_valid(form)` saves the object? Why we are saving it manually? I think we don't need to overwrite this method. See https://docs.djangoproject.com/en/4.1/ref/class-based-views/mixins-editing/#django.views.generic.edit.ModelFormMixin.form_valid

Reply:
This is a `FormView`, not a `ModelFormView`